internalsigner.go

```diff
@@ -23,14 +23,13 @@ func (is *InternalSigner) Sign(index uint32, unsignedTx *Tx) (signedTx *Tx, err
 		is.SigHashFlag = sighash.AllForkID
 	}
 
-	sh, err := unsignedTx.GetInputSignatureHash(index, is.SigHashFlag)
-	if err != nil {
+	var sh []byte
+	if sh, err = unsignedTx.GetInputSignatureHash(index, is.SigHashFlag); err != nil {
```

Comment:
@mrz1836 does your IDE automatically do this?

Reply:
Yeah my IDE complains, [shadow var](https://yourbasic.org/golang/gotcha-shadowing-variables/) again... overwriting `err`